styx-scheduler-service/src/test/java/com/spotify/styx/state/handlers/FlyteRunnerHandlerTest.java

```diff
@@ -165,7 +155,24 @@ public void shouldNotThrowExceptionIfEvenRouterIsClosed(State state) throws Exce
         .build());
 
     flyteRunnerHandler.transitionInto(runState, eventRouter);
+  }
+
+  @Test
+  public void shouldPollInRunning() throws FlyteRunner.PollingException {
+    RunState runState = RunState.create(WORKFLOW_INSTANCE, State.RUNNING, StateData.newBuilder()
+        .executionId(EXECUTION_ID)
+        .executionDescription(FLYTE_EXECUTION_DESCRIPTION)
+        .build());
+
+    flyteRunnerHandler.transitionInto(runState, eventRouter);
+
+    verify(flyteRunner).poll(getFlyteExecutionId(FLYTE_EXECUTION_DESCRIPTION, EXECUTION_ID), runState);
```

Comment:
The test is failing because `FlyteExecutionId` doesn't override `equals` and `hashCode` also it would be nice to override `toString`. Actually what it makes more sense is to convert `FlyteExecutionId` to AutoValue or Automatter

Reply:
yes i will fix